pquery3-run.conf

```diff
@@ -129,9 +130,9 @@ PQUERY_RUN_TIMEOUT=100
 # so on...                                                                     #
 # ex. if PQUERY_RUN_TIMEOUT=100 and TIMEOUT_INCREMENT=10                       #
 # Trial1=100, Trial2=110s, Trial3=120s                                         #
-# To disable set TIMEOUT_INCREMENT=0                                           #
+# To disable set TIMEOUT_INCREMENT=0(default)                                  #
 ################################################################################
-TIMEOUT_INCREMENT=10
```

Comment:
why you set TIMEOUT_INCREMENT default to 0? let it be more than 0 in default

Reply:
In QA, normally the runs are done overnight with TRIALS=1000. This has been the way since pquery2. If we not set TIMEOUT=0 (default), people may not notice it. This will cause each trial to be delayed and later trials will have 1000sec delay.